### PR TITLE
v2.18.0

### PR DESCRIPTION
**Pull request by amortemousque**
---

I have gone over the [contributing](https://github.com/DataDog/browser-sdk/blob/main/CONTRIBUTING.md) documentation.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +16,12 @@
 
 ---
 
+## v2.18.0
+
+- 👷 Versioned CDN files ([#932](https://github.com/DataDog/browser-sdk/pull/932))
+- 👷 Increase browser cache to 4 hours ([#933](https://github.com/DataDog/browser-sdk/pull/933))
+- 📝 add `actionNameAttribute` initialization parameter ([#923](https://github.com/DataDog/browser-sdk/pull/923))
```
Comment (bcaudan) [inline CHANGELOG.md:23]: did you remove `🐛 ensure that test cookie is correctly deleted after the test (#927)`?

Comment (amortemousque) [reply CHANGELOG.md:23]: My bad, I thought it was only for tests...

Comment (amortemousque) [reply CHANGELOG.md:23]: I'll add it. Are the 3 other points good for you?

Comment (bcaudan) [reply CHANGELOG.md:23]: LGTM!

**Pull request closed without merging**